htdocs/user/card.php

```diff
@@ -772,7 +772,19 @@
 	$formproduct = new FormProduct($db);
 }
 
-if ($object->id > 0) {
+if (GETPOSTINT('id') > 0) {
```

Comment:
The fetch is using $id so the test should be on $id

Reply:
Hello @eldy,

We forced to use "**GETPOSTINT('id')**" instead of "**$id**".

Because there is a test at top that checks if the "**$id**" is empty :

`if (empty($id)) {`
	`$id = $user->id;`
`}`

![image](https://github.com/Dolibarr/dolibarr/assets/81772051/ec06d235-1bf0-4cfa-ba16-e4f390da0ad8)

The condition "($id > 0)" will be always true. So, when we create a user, the page title will be the name of the current user and this is wrong.

We have changed the structure now. Could you check it from your side?

Best regards